src/renderer/hooks/usePaneNodeSearchMenu.tsx

```diff
@@ -379,7 +380,7 @@ export const usePaneNodeSearchMenu = (
                     },
                     nodeType: schema.nodeType,
                 },
-                stoppedOnIterator || undefined
+                stoppedOnIterator || parent || undefined
```

Comment:
The `|| undefined` is unnecessary, I think.

Reply:
Since it was there before I'd rather not remove that unless you're sure

Reply:
I am sure. The reason it was there in the first place is that `stoppedOnIterator` allows `null` (for some reason), but the parameter expected `string | undefined`.

Reply:
Also, with `null` and `undefined` in general: TypeScript is really good at tracking these. If TS says it's fine, then it most likely is.